classes/script_metadata.php

```diff
@@ -40,8 +40,14 @@ class script_metadata {
         manager::FLAME_OFF_PARAM_NAME,
     ];
 
-    /** List of paramteres that are to be recorded in redacted form. */
-    const REDACTLIST = ['sesskey'];
+    /** List of parameters that are to be recorded in redacted form. */
+    const REDACTLIST = [
+        'sesskey',
+        'wstoken',
+        'authtoken',
+        'key',
+        'nonce',
```

Comment:
Could we sort this list? It would be quicker to look things up if things were alphabetical.